frontend/packages/dev-console/src/components/project-access/__tests__/ProjectAccess.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { Formik } from 'formik';
+import { LoadingBox, StatusBox } from '@console/internal/components/utils';
+import ProjectAccess, { ProjectAccessProps } from '../ProjectAccess';
+
+const projectAccessProps: ProjectAccessProps = {
+  formName: 'project access',
+  namespace: 'abc',
+  roleBindings: {
+    data: [],
+    loaded: true,
+    loadError: {},
+  },
+};
+
+describe('Project Access', () => {
+  let renderProjectAccess: ShallowWrapper<ProjectAccessProps>;
```

Comment:
No need for the declaration to be here. Can leave it inside the individual tests.